lib/compiler/test_runner.zig

```diff
@@ -12,10 +12,8 @@ var cmdline_buffer: [4096]u8 = undefined;
 var fba = std.heap.FixedBufferAllocator.init(&cmdline_buffer);
 
 pub fn main() void {
-    if (builtin.zig_backend == .stage2_riscv64) return mainExtraSimple() catch @panic("test failure");
-
-    if (builtin.zig_backend == .stage2_aarch64) {
```

Comment:
What happened to this prong?

Reply:
As far as I can tell the `aarch64` backend is so bit-rotten that it cannot even run the `mainSimple` test runner. When work resumes I'm sure there'll be a similar `mainExtraSimple` function and whoever is working on it can change this as needed.

Reply:
Makes sense!